tests/runtests.py

```diff
@@ -510,6 +510,26 @@ def test_cmd_xor_memory(self):
         self.assertIn("Patching XOR-ing ", res)
         return
 
+    def test_cmd_highlight(self):
+
+        cmds = [
+            "highlight add 41414141 yellow",
+            "highlight add 42424242 blue",
+            "highlight add 43434343 green",
+            "highlight add 44444444 pink",
+            'set $rsp = "AAAABBBBCCCCDDDD"',
+            "hexdump qword $rsp 2"
+        ]
+
+        res = gdb_start_silent_cmd(cmds[0], after=cmds[1:], strip_ansi=False)
```

Comment:
Why `cmds[1:]`?

Reply:
This was a hack because the first argument doesn't take an array of commands.  I suppose another hack could be setting the first command to `''`, and `after` to `cmds`.